tools/listProtos.ts

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
```

Comment:
Wouldn't this require a change to `tsconfig.json` to actually compile the file?

Reply:
It actually works :) TypeScript unofficially supports it (https://github.com/microsoft/TypeScript/pull/4120). I actually copied this from another script in `tools/` that also works.

Reply:
lol wow, TIL - though maybe being explicit here would be a good thing 😆